packages/rule-doc-generator/tests/Text/KeywordHighlighterTest.php

```diff
@@ -36,6 +36,7 @@ public function provideData(): Iterator
         yield ['some @var text', 'some `@var` text'];
         yield ['@param @var text', '`@param` `@var` text'];
         yield ['some @var and @param text', 'some `@var` and `@param` text'];
+        yield ['Split multiple inline assigns to each own lines default value, to prevent undefined array issues', 'Split multiple inline assigns to each own lines default value, to prevent undefined array issues'];
```

Comment:
This case should be covered with quoting:

```php
yield [
    'Run function each() on every line', 
    'Run function `each()` on every line'
],
```

Reply:
that already in next example for has `()`:

```php
      yield [
             'autowire(), autoconfigure(), and public() are required in config service',
             '`autowire()`, `autoconfigure()`, and `public()` are required in config service',
```

Reply:
I see :+1: